src/components/Todo/Todo.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { User } from '../User';
+import { todoShape } from '../../shapes';
+
+export const Todo = ({ todoObj }) => (
```

Comment:
don't use 'obj' in variable name
```suggestion
export const Todo = ({ todo }) => (
```